data/map planets.txt

```diff
@@ -286,8 +286,8 @@ planet Babiali
 planet Baianus
 	attributes frozen uninhabited
 	landscape land/snow20
-	description `This frozen water world is reminiscent of Titan back in Sol: the surface is little more than a shattered wasteland of ice and the occasional rock outcropping, but the unbreathably thin atmosphere contains countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
-	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
+	description `This frozen water world is reminiscent of Triton back in Sol: the surface is little more than a shattered wasteland of ice and the occasional rock outcropping, but the unbreathably thin atmosphere contains countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
+	description `	Unlike Triton, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```

Comment:
@Zitchas suggested restructuring of the information for clarity:

```suggestion
	description `From a distance this large, icy moon is superficially reminiscent of Titan back in Sol: the surface is little more than a shattered wasteland of ice and the occasional rock outcropping, it is similarly exposed to it's giant parent's radiation belts, and scans indicate countless tantalizing traces of chemicals that suggest that something more might exist below the surface.`
	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras swirl through it's unbreathably thin atmosphere, accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
```

Reply:
From a distance in stellar terms, it'd be about the same mass, orbiting in a relatively similar range.
Those are the first two things you can tell from a distance with the least amount of detection equipment.

Sure the *next* thing you'd notice is the difference in atmosphere which is very different, but I don't think the point is invalid.

... Europa might be a better match up close though, albeit one that is substantially smaller than Titan.

Reply:
Personally, mass is the main thing I knew about Titan, but the principle of this suggestion was just to restructure Zitchas's original intent along the lines Derpy proposed for aiding comprehension.

If you really want to debate for a different stellar body entirely, you can have that debate with someone else.

Reply:
I'm good with MasterOfGrey's proposed changes here.

Reply:
> From a distance in stellar terms, it'd be about the same mass, orbiting in a relatively similar range.
Those are the first two things you can tell from a distance with the least amount of detection equipment.

I don't think this explanation lines up with the proposed description very well.
If size and orbital distance are the similarities the opening line of the description alludes to, it seems odd to elaborate on that statement of superficial reminiscence by describing the surface features. Any similarities in the surface would only be visible past the differences in the atmospheres, and, at the point of considering the atmospheric differences, it's difficult to say calling "reminiscent of Titan," is a reasonable description.
The description of the surface features also ignores perhaps the most unique property of Titan's surface, it's hydrocarbon lakes, and focuses on properties that are common to a number of other moons in our system.

Reply:
I have no strong feelings either way. If you really want to change it, go for it. I don't think it improves anything, but I don't think it makes anything worse, either. Unnecessary, but neutral.

Reply:
I have updated the above suggestion with deference to Warp's notes.

Reply:
Trust looks good.